samples/net/sockets/can/Kconfig

```diff
@@ -0,0 +1,15 @@
+# Private config options for can sample app
+
+# Copyright (c) 2018 Alexander Wachter
+# SPDX-License-Identifier: Apache-2.0
+
+mainmenu "SocketCAN sample application"
+
+config LOOPBACK_MODE
```

Comment:
For the sample specific Kconfig options in the networking samples, I added `SAMPLE` to the option name in order to make it obvious that this option is strictly sample specific. So I am suggesting to call this `CONFIG_SAMPLE_CAN_LOOPBACK_MODE` here?

Reply:
Done